docs/trusted-resources.md

```diff
@@ -54,19 +54,21 @@ metadata:
     app.kubernetes.io/instance: default
     app.kubernetes.io/part-of: tekton-pipelines
 data:
-  resource-verification-mode: "enforce"
+  trusted-resources-verification-no-match-policy: "fail"
 ```
 
-**Note:** `resource-verification-mode` needs to be set as `enforce` or `warn` to enable resource verification.
+`trusted-resources-verification-no-match-policy` configurations:
+ * `ignore`: Don't fail the taskrun/pipelinerun if no matching policies are found. Skip the verification and don't log.
+ * `warn`: Don't fail the taskrun/pipelinerun if no matching policies are found. Skip the verification and log a warning
```

Comment:
```suggestion
 * `ignore`: if no matching policies are found, skip the verification, don't log, and don't fail the taskrun/pipelinerun
 * `warn`:  if no matching policies are found, skip the verification, log a warning, and don't fail the taskrun/pipelinerun
```

Just trying to make it extra clear that verification is only skipped when no matching policies are found

Reply:
Thank you!